examples/connext_dds/compression/c++11/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# (c) 2021 Copyright, Real-Time Innovations, Inc. All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+cmake_minimum_required(VERSION 3.11)
+project(rtiexamples-compression)
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
+)
+
+# Include ConnextDdsAddExample.cmake from resources/cmake
+include(ConnextDdsAddExample)
+
+find_package(RTIConnextDDS
```

Comment:
I think it isn't necessary, we are already calling this function from resources/cmake/ConnextDdsAddExample.cmake. This can be removed because if we update the Connext version, we should modify many files.